model/system/metrics.yaml

```diff
@@ -430,3 +430,95 @@ groups:
       - ref: linux.memory.slab.state
     entity_associations:
       - host
+
+  - id: metric.system.linux.mdraid.disks
+    type: metric
+    metric_name: system.linux.mdraid.disks
+    stability: development
+    brief: "Number of active/failed/spare disks of device."
+    note: |
+      Reports the total number of disks in a linux RAID device.
+      See also [mdadm](https://raid.wiki.kernel.org/index.php/Mdstat).
+    instrument: gauge
+    unit: "By"
```

Comment:
The `By` unit is "bytes", which is not the correct unit for this metric. For this metric it would likely be `{disk}`. I see that each of the metrics here use the `By` unit. I'm not sure any of these are tracking bytes, so the same adjustment will need to be made to each metrics's unit.

Reply:
changed to {disk}